src/ChainSafe.GamingSDK.EVM/Web3/Core/Analytics/ApiAnalyticsClient.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Generic;
+using ChainSafe.GamingWeb3.Environment;
+
+namespace ChainSafe.GamingWeb3.Analytics
+{
+    public class ApiAnalyticsClient : IAnalyticsClient
+    {
+        private const string LoggingUrl = "https://game-api-stg.chainsafe.io/logging/logEvent";
```

Comment:
Why are we using the staging API here? AFAIK, the API is also available on production, so let's use that.